code/modules/client/preference/preferences_mysql.dm

```diff
@@ -440,7 +440,7 @@
 													"f_style" = f_style,
 													"markingstyleslist" = markingstyleslist,
 													"ha_style" = ha_style,
-													"alt_head" = alt_head || "",
```

Comment:
Does this not actually work for a null `alt_head`? That seems odd.

Reply:
Nope. It doesnt.